### DISCUSSION TITLE
Add a dismiss for Tip/Warning UI Elements

### DISCUSSION THREAD

**Discussion by john-henry** [craftcms/cms#12188] [Ideas] [answered]
Over time a Tip or warning can become annoying to a user who has already seen them. The request is to make these dismissible per user. Added benefit of freeing up space on the entry form.

**Comment by brandonkelly** [comment, marked as answer]
We’ve made this possible for Craft 4.4, via #12238 🎉

**Reply by brandonkelly** [reply]
4.4 is out! Thanks again for the suggestion @john-henry!

**Comment by leevigraham** [comment]
@brandonkelly @i-just Is there anyway to restore the ui elements after the user has dismissed? Maybe a button in preferences to clear local storage?

**Reply by brandonkelly** [reply]
Not currently. You could force them to show back up for all users by recreating the tip/warning from the field layout designer.
